### PR TITLE
Fix/#14 add comment to function

### LINKED ISSUE #14: Add Documentation Comment to Public Function
## Description

Add a documentation comment to the public function in order to improve code readability and facilitate future maintenance.

## Requirements

- Identify the public function that requires a documentation comment.
- Write a documentation comment in Markdown format above the function declaration.
- The documentation comment should include the following sections:
  - **Description**: A brief summary of the function's purpose and functionality.
  - **Arguments**: List and describe each input parameter, including its type and any additional details.
  - **Return**: Explain the return value or specify if the function returns `void`.
  - **Example(Option)**: Provide an example code snippet demonstrating the usage of the function if applicable.
  - **Notes(Option)**: Include any additional notes or considerations that may be useful for developers.

## Example

```Rust
/// Adds two numbers together and returns the result.
///
/// # Arguments
/// * `a` - The first number.
/// * `b` - The second number.
///
/// # Returns
/// The sum of `a` and `b`.
/// 
pub fn add_numbers(a: i32, b: i32) -> i32 {
    a + b
}
```

### PR DESCRIPTION
**Pull request by sheagrief**
fix #14

### CODE REVIEW

**Comment by sheagrief** [comment]
#14

File: src/she.rs [hunk 1]
```diff
@@ -16,6 +16,12 @@ pub use plaintext::{Plaintext, Plaintextish, Plaintexts};
 use rand::Rng;
 use rand_distr::{Distribution, Normal};
 
+/// Parameters for ZKPoPK.
+/// s: length of Plaintext.
+/// n: degree of polynomial (length of Encodedtext) should be same with lenght of Plaintext.
```
Comment (taskooh) [inline src/she.rs:21]:
```suggestion
/// n: degree of polynomial (length of Encodedtext) should be same with length of Plaintext.
```

File: src/she/polynomial.rs [hunk 2]
```diff
@@ -65,6 +83,14 @@ pub fn substitute<F: Field>(polynomial: &[F], variable: &F) -> F {
...
+/// Compute the roots of the cyclotomic polynomial \Phi_N(X) on F. wbere N is expected to be a power of two.
+///
+/// # Arguments
+/// * `length` - The length of the roots.
+///
+/// # Returns
+/// The roots of the cyclotomic polynomial.
+///
 pub fn cyclotomic_moduli<F: FftField>(length: usize) -> Vec<F> {
     // moduli: lengthは本来N-1だが、sで切り捨て
```
Comment (taskooh) [inline src/she/polynomial.rs:95]: Can you translate here to english?

File: src/she/polynomial.rs [hunk 3]
```diff
@@ -65,6 +83,14 @@ pub fn substitute<F: Field>(polynomial: &[F], variable: &F) -> F {
     result
 }
 
+/// Compute the roots of the cyclotomic polynomial \Phi_N(X) on F. wbere N is expected to be a power of two.
```
Comment (taskooh) [inline src/she/polynomial.rs:86]:
```suggestion
/// Compute the roots of the cyclotomic polynomial \Phi_N(X) on F. where N is expected to be a power of two.
```

File: src/she/polynomial.rs [hunk 4]
```diff
@@ -7,6 +7,15 @@ use ark_poly::{
...
 
+/// Interpolate polynomials such that the result of evaluated on each point is each number.
+///
+/// # Arguments
+/// * `eval_at` - The value to evaluate.
+/// * `evals` - The number of the evaluation.
+///
+/// # Returns
+/// The interpolated polynomial.
+///
```
Comment (taskooh) [inline src/she/polynomial.rs:18]:
```suggestion
/// Interpolate a polynomial such that it passes through specified points.
/// This function calculates an interpolated polynomial that passes through a set of given points.
///
/// # Arguments
///
/// * `eval_at` - A vector containing the x-values of the points to interpolate.
/// * `evals` - A vector containing the corresponding y-values of the points to interpolate.
///
/// # Returns
/// An `Option` containing the coefficients of the interpolated polynomial. Returns `None` if interpolation fails.
```

**Review by taskooh — CHANGES_REQUESTED** [review]

**Pull request merged**